README.md

```diff
@@ -2,12 +2,6 @@
 
 Thorn is composed of a set of helper abstractions built to ease the process of setting up a [SugarCRM](https://www.sugarcrm.com) instance's REST API testing environment and interacting with it.
 
-## Installation
```

Comment:
@rhoggSugarcrm why not keeping at least a reference to installation steps here?

Reply:
Restored!